editor/src/components/canvas/canvas-strategies/strategies/resize-grid-strategy.ts

```diff
@@ -70,7 +70,13 @@ export const resizeGridStrategy: CanvasStrategyFactory = (
     return null
   }
 
-  const gridPath = isGrid ? selectedElement : EP.parentPath(selectedElement)
+  const gridPath = isGrid
+    ? selectedElement
+    : findOriginalGrid(canvasState.startingMetadata, EP.parentPath(selectedElement)) // TODO don't use EP.parentPath
```

Comment:
findOriginalGrid is called again a few lines below (and for the note, not really the scope of this PR, but we should not need this to get the grid templates, but use `parentContainerGridPropertiesFromProps` on the grid item

Reply:
[`4587edd` (#6606)](https://github.com/concrete-utopia/utopia/pull/6606/commits/4587edd1d83796afbe10e43ebe9f72a46f5a6cce)